### ISSUE TITLE
Add bullet penetration

### ISSUE THREAD

**Issue by malcool3** [LFalch/tds#17]
allow for bullets of certain calibers to pass through certain materials

**Comment by LFalch** [comment]
Do we need to add a hardness property or something like that to each solid material?

**Comment by malcool3** [comment]
yeah, bricks being harder than wood, glass being very brittle and such, defining calibers shouldn't be harder than adjusting penetration, perhaps having a certain number for each caliber.
